lib/oli/delivery/sections/section_cache.ex

```diff
@@ -81,14 +127,51 @@ defmodule Oli.Delivery.Sections.SectionCache do
   end
 
   def handle_call({:put, key, value}, _from, state) do
-    ttl = :timer.hours(24 * 365)
-
-    case Cachex.put(@cache_name, key, value, ttl: ttl) do
+    case Cachex.put(@cache_name, key, value, ttl: @ttl) do
       {:ok, true} ->
         {:reply, :ok, state}
 
       _ ->
         {:reply, :error, state}
     end
   end
```

Comment:
Does it make sense to use `handle_cast` for `put` and `delete`? For `delete` Cachex docs say "This will return { :ok, true } regardless of whether a key has been removed or not"